lang/es_ES.json

```diff
@@ -384,11 +384,13 @@
     "players are currently online.": "jugadores actualmente en línea.",
     "players in-game.": "jugadores echando una partida.",
     "supportedGameFilesCountLabel_one": "Actualmente <1>{{count, number}}</1> registro de hash de archivo soportado registrado para este juego.",
+    "supportedGameFilesCountLabel_many": "Actualmente hay registrados <1>{{count, number}}</1> hashes de archivos soportados registrados para este juego.",
```

Comment:
Your description specifically mentions `_few` keys missing, but you didn't add any. Was that intentional?
> When there are 2 supported game files and the user is using pl_PL, i18next is naturally reaching for the "_few" key, but it currently isn't present, causing that string to fall back to English.

Reply:
The script only adds them in the locales where they're needed, such as the pl_PL file:

![Screenshot 2024-11-15 at 7 21 57 AM](https://github.com/user-attachments/assets/0c53545e-dd19-4564-93c9-a4f9a0d0c06b)

i18next is smart enough to know based on the locale code what plural forms are expected of the locale. This is why the file rename before & after the sync script is necessary, so the locale detection can work correctly.